src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -52,7 +52,7 @@ public PersonCard(Person person, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
         phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
+        telegramHandle.setText(person.getTelegramHandle().value);
```

Comment:
tbh i guess we could prepend @ to the front of the value just so on the ui it looks like an actual tele handle

Reply:
sounds like a good solution